src/OmniSharp/Options/FormattingOptions.cs

```diff
@@ -2,6 +2,15 @@ namespace OmniSharp.Options
 {
     public class FormattingOptions 
     {
+        public FormattingOptions()
+        {
+            //just defaults
```

Comment:
the defaults I here because when we now flow the options into formatter, it was using the default state of this object - so indentation was 0 which was causing some tests to fail